examples/vpolytope-volume/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# VolEsti (volume computation and sampling library)
```

Comment:
is this file needed?

Reply:
Yes it contains some cmake instructions
```add_executable (vpolytopevolume vpolytopevolume.cpp)
TARGET_LINK_LIBRARIES(vpolytopevolume ${LP_SOLVE})```

Reply:
but you can use the cmake file from `examples` folder to build this example

Reply:
OK implementing now